core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -672,5 +691,20 @@ protected void runCatchup() {
       // Update suspended sequence after catch-up is finished.
       mSuspendSequence = catchUp(mCatchUpStartSequence, mCatchUpEndSequence) - 1;
     }
+
+    @Override
+    public void onError(Throwable t) {
+      if (ExceptionUtils.containsInterruptedException(t)) {
```

Comment:
But this is assuming we correctly shutdown by interrupting threads. Otherwise extra crashes await :)